provider/equinix/environ.go

```diff
@@ -302,20 +264,52 @@ func (e *environ) StartInstance(ctx context.ProviderCallContext, args environs.S
 		return nil, errors.Trace(err)
 	}
 
-	if err := e.finishInstanceConfig(&args, spec); err != nil {
-		return nil, errors.Trace(err)
-	}
+	return spec, nil
+}
 
+func getCloudConfig(args environs.StartInstanceParams) (cloudinit.CloudConfig, error) {
 	cloudCfg, err := cloudinit.New(args.InstanceConfig.Base.OS)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	cloudCfg.AddPackage("iptables-persistent")
+
+	// Set a default INPUT policy of drop, permitting ssh
+	acceptInputPort := "iptables -A INPUT -p tcp --dport %d -j ACCEPT"
+	iptablesDefault := []string{
+		"iptables -A INPUT -m conntrack --ctstate INVALID -j DROP",
+		"iptables -A INPUT -m conntrack --ctstate ESTABLISHED,RELATED -j ACCEPT",
+		"iptables -A OUTPUT -m conntrack --ctstate ESTABLISHED -j ACCEPT",
+		"iptables -A INPUT -p icmp -j ACCEPT",
+		"iptables -A INPUT -i lo -j ACCEPT",
+		"iptables -A OUTPUT -o lo -j ACCEPT",
+		"iptables -P INPUT ! -i lo -s 127.0.0.0/8 -j REJECT",
+		fmt.Sprintf("iptables -A OUTPUT -p tcp --sport %d -m conntrack --ctstate ESTABLISHED -j ACCEPT", ssh.SSHPort),
```

Comment:
I think we should fix this PR up before bringing it across. I don't like that a provider has a double reference into the cmd code.

We should just make this a const in the package.

Reply:
i've extracted a const for the iptables commands